content/blog/kubernetes-chart-v4/index.md

```diff
@@ -0,0 +1,846 @@
+---
+title: "New: Helm Chart resource for Java, YAML SDKs"
+date: 2024-04-12T00:00:00-07:00
+meta_desc: >-
+  Pulumi Kubernetes v4.13 offers a new resource for applying Helm charts consistently across Pulumi SDKs, and broadens support to the Java and YAML SDKs.
+
+authors:
+    - eron-wright
+
+tags:
+    - kubernetes
+    - yaml
+    - java
+    - helm
+---
+
+When you need to install a third-party application into your Kubernetes cluster, you're likely to find a
+Helm chart for that in [Artifact Hub](https://artifacthub.io/) or other registry. Pulumi provides two
+ways to apply a Helm chart, as outlined in [Choosing the Right Helm Resource For Your Use Case](/registry/packages/kubernetes/how-to-guides/choosing-the-right-helm-resource-for-your-use-case/).  The Chart resource offers deeper integration with Pulumi
```

Comment:
Are we updating that how to guide to include v4?

Reply:
Yes, already done.